acme/errors.go

```diff
@@ -75,6 +75,8 @@ func (ap ProblemType) String() string {
 		return "accountDoesNotExist"
 	case ErrorAlreadyRevokedType:
 		return "alreadyRevoked"
+	case ErrorBadAttestationStatementType:
+		return "badAttestationStatement"
```

Comment:
This list of errors is based on the list at https://datatracker.ietf.org/doc/html/rfc8555#section-6.7. If we add this one, it probably also needs to end up in the RFC.

Reply:
https://github.com/brandonweeks/draft-bweeks-acme-device-attest/issues/5